KeyboardHook.cs

```diff
@@ -40,7 +41,12 @@ private IntPtr HookCallback(int nCode, IntPtr wParam, IntPtr lParam)
                     try
                     {
                         System.Threading.Thread.Sleep(killerSpeed);
-                        Process.Start(GameLauncherPath);
+                        Process proc = Process.Start(GameLauncherPath);
+                        if (PriorityAffinity)
+                        {
+                            proc.ProcessorAffinity = (System.IntPtr)0x08;
+                            proc.PriorityClass = ProcessPriorityClass.RealTime;

```

Comment:
I believe we should use high priority instead of real-time unless you have some specific reason as to why real-time is more appropriate.

Reply:
There were reports that the game crashed way less with real-time priority compared to normal, although there wasn't much testing done with high. Is there any reasoning behind avoiding real-time?

Reply:
Would be cool to have some testing for "high" first then. Would prefer high to real-time because of the concerns raised here https://devblogs.microsoft.com/oldnewthing/20100610-00/?p=13753 and here https://stackoverflow.com/questions/1663993/what-is-the-realtime-process-priority-setting-for

Reply:
Since I am not familiar with what each setting exactly does to make the game consistent, I will consider adding an option to select multiple priorities and affinities, I think that will be better.